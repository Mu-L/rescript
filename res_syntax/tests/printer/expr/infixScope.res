
module M = {
    @@infix.add(("+++", "+"))

    let z = 3 + 4
}

let z = 3 + 4