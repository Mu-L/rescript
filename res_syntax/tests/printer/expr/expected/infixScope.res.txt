module M = {
  @@infix(("+++", "+"))

  let z = 3 +++ 4
}

let z = 3 + 4
